codec-http2/src/main/java/io/netty/handler/codec/http2/HpackDecoder.java

```diff
@@ -341,27 +326,18 @@ public void setMaxHeaderTableSize(long maxHeaderTableSize) throws Http2Exception
         }
     }
 
-    public void setMaxHeaderListSize(long maxHeaderListSize, long maxHeaderListSizeGoAway) throws Http2Exception {
```

Comment:
@ejona86 should we just keep the method as well and deprecate it and ignore the second argument ?

Reply:
Unclear how many people are using that, but sure. It's easy. Done.